tools/travis/flake8.sh

```diff
@@ -14,7 +14,7 @@ declare -a PYTHON_FILES=("."
 echo 'Flake8: first round (fast fail) stops the build if there are any Python 3 syntax errors...'
 for i in "${PYTHON_FILES[@]}"
 do
-    flake8 "$i" --select=E999 --statistics
+    flake8 "$i" --select=E999,F821 --statistics
```

Comment:
There are a [ton of rules in flake8](https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes) and as was discussed in https://github.com/openwhisk/openwhisk/pull/2056 we do not want to get too nitpicky in this project where Python is just 1.7% of the codebase.  I think the two pass compromise that we settled on is the right one.  Code issues that halt the runtime should also halt PR builds but those code issues that do not halt the runtime are recorded for those that care to review them but do not halt progress.